### PR TITLE
Release OCaml Jupyter kernel v2.2.1

### PR DESCRIPTION
**Pull request by akabe**
Release note: https://github.com/akabe/ocaml-jupyter/releases/tag/v2.2.1

### CODE REVIEW

**Comment by djs55** [comment]
CentOS 7 failed because `conf-zmq` is missing a `depext`. That's a separate problem.

The `archimedes` package failed with
```
# File "src/archimedes_graphics.ml", line 97, characters 6-24:
# Error: Unbound module Graphics
```
which looks like a separate problem too.

File: packages/jupyter/jupyter.2.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "uuidm" {>= "0.9.6"}
+  "base64" {>= "2.1.2"}
+  "lwt" {>= "3.0.0"}
+  "stdint" {>= "0.4.2"}
+  "zmq" {>= "4.0-8"}
+  "lwt-zmq" {>= "2.1.0"}
+  "yojson" {>= "1.3.3"}
+  "ppx_deriving_yojson" {>= "3.0"}
+  "cryptokit" {>= "1.12"}
+  "ocamlfind" {build & >= "1.5.0"}
```
Comment (djs55) [inline packages/jupyter/jupyter.2.2.1/opam]: I notice the package uses `jbuilder`. Is `ocamlfind` still a dependency? It might be safe to remove now.

**Comment by akabe** [comment]
@djs55 Thanks for your advice. I removed the line you mentioned.

I think `Graphics` module is not compiled because libx11 is not installed in the Docker images: https://github.com/ocaml/opam-repository/pull/10876#issuecomment-347796651

**Comment by djs55** [comment]
Thanks! Looks good to me.

**Pull request merged**